ipysurfer/stc.py

```diff
@@ -0,0 +1,200 @@
+from mne.source_estimate import SourceEstimate
+from mne.utils import _check_subject
+from mne.viz._3d import _handle_time, _limits_to_control_points
+
+from ._utils import _get_subjects_dir
+from .viz import Brain, TimeViewer
+
+
+def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
+                          colormap='auto', time_label='auto',
+                          smoothing_steps=10, transparent=None, alpha=1.0,
+                          time_viewer=False, subjects_dir=None, figure=None,
+                          views='lat', colorbar=False, clim='auto',
+                          cortex='classic', size=800, background='black',
+                          foreground=None, initial_time=None, time_unit='s'):
+    u"""Plot SourceEstimates with ipyvolume.
+
+    Parameters
+    ----------
+    stc : SourceEstimates
+        The source estimates to plot.
+    subject : str | None
+        The subject name corresponding to FreeSurfer environment
+        variable SUBJECT. If None stc.subject will be used. If that
+        is None, the environment will be used.
+    surface : str
+        The type of surface (inflated, white etc.).
+    hemi : str, 'lh' | 'rh' | 'split' | 'both'
+        The hemisphere to display.
+    colormap : str | np.ndarray of float, shape(n_colors, 3 | 4)
+        Name of colormap to use or a custom look up table. If array, must
+        be (n x 3) or (n x 4) array for with RGB or RGBA values between
+        0 and 255. Default is 'hot'.
+    time_label : str | callable | None
+        Format of the time label (a format string, a function that maps
+        floating point time values to strings, or None for no label). The
+        default is ``time=%0.2f ms``.
+    smoothing_steps : int
+        The amount of smoothing
```

Comment:
could we make this more descriptive? "amount of smoothing" is a bit vague - what exactly does it correspond to? Alternatively if this is a copy of the docstring from the MNE version, we can leave it

Reply:
Yes, it's a copy from `MNE` function. I will leave like that, ok?

Reply:
yep - and if any of my other comments are equally relevant to the MNE docstrings, I don't think they need to be fixed here